src/components/app/game-board/side-panel/game-status/ProgressBar/ProgressBar.module.scss

```diff
@@ -0,0 +1,8 @@
+/* ProgressBar.module.scss */
```

Comment:
Do we need this? And please consider to remove the world 'module' from the file name

Reply:
I thought this is the standard. Good to know.